src/test/javascript/spec/integration/code-editor/code-editor-container.integration.spec.ts

```diff
@@ -556,8 +545,8 @@ describe('CodeEditorContainerIntegration', () => {
         isCleanSubject.next({ repositoryStatus: CommitState.CONFLICT });
         containerFixture.detectChanges();
 
-        expect(container.commitState).to.equal(CommitState.CONFLICT);
-        expect(getRepositoryContentStub).to.not.have.been.called;
+        expect(container.commitState).toEqual(CommitState.CONFLICT);
+        expect(getRepositoryContentStub).not.toHaveBeenCalled;
```

Comment:
```suggestion
        expect(getRepositoryContentStub).not.toHaveBeenCalled();
```
Same as above (The function call is omitted).

Reply:
Same.